### ISSUE TITLE
Problem: Normalization job for AIP and DIP requires user input despite processing configuration settings

### ISSUE THREAD

**Issue by lwo** [archivematica/Issues#410]
**Expected behaviour**
The ingest continues without the need of any human to approve the jobs.

**Current behaviour**
When an AIP and DIP are part of the normalization procedure; and the Processing configuration is all set to approve all steps automatically, the ingest UI requires human user approval.

**Steps to reproduce**
Set the default config to:
Normalize: "Normalize for preservation and access"
Approve normalization: "Yes"

Then select a transfer with "Approve automaticaly" enabled.
Run the Transfer.
On the Ingest tab, we see the process continue until job "Nomalization". We see a question appear:
Job: Normalize [?]: awaiting decision. And the popup gives three choices: normalize for preservation; reject SIP and Do not normalize.

**Your environment (version of Archivematica, OS version, etc)**
Ubuntu 16.04 
Fork: https://github.com/IISH/archivematica

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by sromkey** [comment]
Hi @lwo - could you provide a sample transfer that we can reproduce this with? Even just the directory structure- it sounds to me like Archivematica might be looking for normalized derivatives or something.

**Comment by lwo** [comment]
Hello,

Yes, the tree structure is:

ARCH00842.1
├── access
│   └── preservation
│       ├── ARCH00842.1_0001.jpg
│       └── ARCH00842.1_0002.jpg
├── metadata
│   ├── checksum.md5
│   ├── identifiers.json
│   └── mets_structmap.xml
└── preservation
    ├── ARCH00842.1_0001.tif
    └── ARCH00842.1_0002.tif

Is that sufficient for a diagnose?

**Comment by sromkey** [comment]
Yes, it looks like Archivematica might be getting confused ;) Is there actually a directory called preservation inside the access folder? Could you try removing that?

**Comment by lwo** [comment]
When I go from the above:

```
ARCH00842.1
├── access
│   └── preservation
│       ├── ARCH00842.1_0001.jpg
│       └── ARCH00842.1_0002.jpg
├── metadata
│   ├── checksum.md5
│   ├── identifiers.json
│   └── mets_structmap.xml
└── preservation
    ├── ARCH00842.1_0001.tif
    └── ARCH00842.1_0002.tif
```
into
ARCH00842.1
└── access
    ├── ARCH00842.1_0001.tif
    └── ARCH00842.1_0002.tif
├── metadata
│   ├── checksum.md5
│   ├── identifiers.json
│   └── mets_structmap.xml
└── preservation
    ├── ARCH00842.1_0001.tif
    └── ARCH00842.1_0002.tif
```

The job cannot find the originals any longer and states:
Job Identify manually normalized files (green)
Job Check for Service directory (green)
Job Check for Access directory (red) because
No corresponding object for: %SIPDirectory%objects/access/ARCH00842.1_0001.jpg
No corresponding object for: %SIPDirectory%objects/access/ARCH00842.1_0002.jpg

So it does expect a parallel structure...

**Comment by sromkey** [comment]
So what you have now is a transfer that has defined access and preservation derivatives but doesn't have any original objects.
To ask another way... what are the files that you are trying to preserve, and what are the files that you are trying to provide access to?

**Comment by lwo** [comment]
>The "preservation" naming in this folder is misleading... We do not make preservation copies. So:
ARCH00842.1/preservation are the original files we want to store as originals
ARCH00842.1/access are our custom access copies so Archivematica does not have to create them for us.
